### ISSUE TITLE
workspace-grid@hernejj : <Ctrl>+<Alt>+<Up>/<Down> shortcuts don't work

### ISSUE THREAD

**Issue by VivienPelletier** [linuxmint/cinnamon-spices-applets#1826]
workspace-grid@hernejj : `<ctrl>+<alt>+<down>/<up>` shortcut don't work.

```
 * Applet version/Build date (to be found in "About.." or in "metadata.json")
Last commit: [dadb2c8b] Fix for cinnamon 3.8 (#1796) 
 * Cinnamon version (cinnamon --version)
# cinnamon --version                                       
Cinnamon 3.8.2
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...)
# uname -a
Linux [...] 4.16.8-1-ARCH #1 SMP PREEMPT Wed May 9 11:25:02 UTC 2018 x86_64 GNU/Linux
 ```

**Notify author of applet**
@hernejj

**Issue**
The shortcuts `<Ctrl>+<Alt>+<Up>` and `<Ctrl>+<Alt>+<Down>` don't work. They do nothing.
Whereas, the shortcuts `<Ctrl>+<Alt>+<Shift>+<Up>` and `<Ctrl>+<Alt>+<Shift>+<Down>` do work. They move current window to the associate workspace.

**Steps to reproduce**
install or update cinnamon, install wokspace-grid applet. Configure it like in this screenshot.
https://s9.postimg.cc/ea13kelwf/workspace_Grid_Config_Window.png

**Expected behaviour**
The shortcuts `<Ctrl>+<Alt>+<Up>` and `<Ctrl>+<Alt>+<Down>` should move the screen to the workspace respectively above or below.

**Other information**
Appear after update of my system.

Thank you for your job, I use this applet everyday.

**Comment by hernejj** [comment]
This bug has already been found and fixed. I'll roll an update sometime in the next few days and post here when it is done. Thanks for your patience :)

**Comment by hernejj** [comment]
Actually it looks like the update process is automatic. Please update your applet in the "Add Applets to Panel" window and try again.

**Comment by VivienPelletier** [comment]
I just install the applet from this repository. The shortcuts `<Ctrl>+<Alt>+<Up>` and `<Ctrl>+<Alt>+<Down>` persist don't work.

**Comment by hernejj** [comment]
I'm sure that updating the applet should fix your problem. Perhaps you've not updated it properly? Here is what you need to do,

Right click on your panel.
Choose "Add applets to the panel"
Click the "Download" tab.
If a dialog box pops up and asks you to update/refresh, say yes.
If you do not get asked to refresh click the refresh button (circular arrows) at bottom of window.
After refresh, locate Workspace Grid applet in the list and click the Update button (single circular arrow).
The button will change to a check mark when the update is complete.
You should now have an up to date version of the applet.
You'll need to restart Cinnamon to get the new applet code to run.

Let me know how it goes. :)

**Comment by VivienPelletier** [comment]
Hi there, I went a bit deeper in the inspection of this problem. I modified the `applet.js` to log some information.

Here is my modified `applet.js` : https://framadrop.org/r/54DmUl4nXT#mqTk63Vfo+OZ3WDnEq1UQrU2XbMJbn6PZPVfDKBl7Qs=

After I restarted Cinnamon, I obtain the both messages that talk about registering up and down key.
When I press `Ctrl` + `Alt` + `Left or Right` I obtain the message with the good binding key.
When I press `Ctrl` + `Alt` + `Up or Down` I obtain no message ! Here is the problem.

I will try to continue to inspect this problem, but It looks like hard to find a accessible documentation of Muffin.
